index.js

```diff
@@ -57,6 +62,7 @@ if (process.env.TRAVIS) {
   event = 'push';
   commit_message = ''; // wercker does not expose commit message
   pull_request_number = ''; // wercker does not expose pull request number
+  pull_request_target_branch = ''; // wercker does not expose pr target branch
```

Comment:
should we leave them as not defined instead of empty string for strict typechecks?

Reply:
I think empty strings are fine for two reasons, I believe env variables are empty strings by default, and it keeps with the existing pattern without introducing breaking changes.

Reply:
valid!